107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -195,18 +196,19 @@ public void jsr107LoaderGetsOverriddenByTemplate() throws Exception {
   public void jsr107StoreByValueGetsOverriddenByTemplate() throws Exception {
     when(jsr107Service.getTemplateNameForCache("cache")).thenReturn("cacheTemplate");
     when(xmlConfiguration.newCacheConfigurationBuilderFromTemplate("cacheTemplate", Object.class, Object.class)).thenReturn(
-        newCacheConfigurationBuilder().add(new OnHeapStoreServiceConfiguration().storeByValue(false))
+        newCacheConfigurationBuilder()
     );
 
     MutableConfiguration<Object, Object> configuration = new MutableConfiguration<Object, Object>();
 
     ConfigurationMerger.ConfigHolder<Object, Object> configHolder = merger.mergeConfigurations("cache", configuration);
 
-    boolean storeByValue = true;
+    boolean storeByValue = false;
```

Comment:
A cleaner way of initialising this var is to use `configuration.isStoreByValue()`